src/common.rs

```diff
@@ -355,6 +355,60 @@ pub struct LoadAvg {
     pub fifteen: f64,
 }
 
+macro_rules! xid {
+    ($(#[$outer:meta])* $name:ident, $type:ty) => {
+        use std::ops::Deref;
+
+        $(#[$outer])*
+        #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
+        pub struct $name(pub(crate) $type);
+
+        impl Deref for $name {
+            type Target = $type;
+
+            fn deref(&self) -> &Self::Target {
+                &self.0
+            }
+        }
+    };
+}
+
+#[cfg(not(target_os = "windows"))]
+pub mod uid {
+    xid!(
+        /// A user id wrapping a platform specific type
+        Uid,
+        libc::uid_t
+    );
+}
+
+#[cfg(target_os = "windows")]
+pub mod uid {
```

Comment:
Same, please remove the wrapping module.

Reply:
Done